galasa-ui/src/utils/3270/get3270Screenshots.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import { downloadArtifactFromServer } from '@/actions/runsAction';
+import { FileNode, TreeNodeData } from '@/utils/functions/artifacts';
+import { CellFor3270, TerminalImage, TerminalImageField } from '@/utils/interfaces/common';
+import pako from 'pako';
+
+export const flattenedZos3270TerminalData: CellFor3270[] = [];
+export const allImageData: TerminalImage[] = [];
+
+export function splitScreenAndTerminal(input: string) {
+  const parts = input.split('-');
+
+  if (parts.length < 2 || !Number.isInteger(parseInt(parts[parts.length - 1]))) {
```

Comment:
It might be worth adding a comment to explain why we're splitting on `-` and what the expected format of the `input` string is.

It's not immediately obvious what `input` actually refers to, can this be a more descriptive parameter?